liblangutil/CharStream.h

```diff
@@ -92,9 +92,9 @@ class CharStream
 	///@}
 
 private:
-	std::string m_source;
-	std::string m_name;
-	size_t m_position;
+	std::string m_source {};
```

Comment:
In my opinion, this is unnecessary clutter and reduces readability. You have the impression that something special is going on while it's not.